plugins/analysis-ukrainian/src/main/java/org/elasticsearch/plugin/analysis/ukrainian/UkrainianAnalyzerProvider.java

```diff
@@ -23,7 +22,7 @@
     public UkrainianAnalyzerProvider(IndexSettings indexSettings, Environment env, String name, Settings settings) {
         super(indexSettings, name, settings);
         analyzer = new XUkrainianMorfologikAnalyzer(
```

Comment:
I think this is something we should fix upstream, it is a pretty clear bug in the API that you can't access the default stopword set

Reply:
I opened https://issues.apache.org/jira/browse/LUCENE-10413

Reply:
@romseygeek This makes sense to me. I will update this PR with a new lucene snapshot, once your changes are merged in 9.x branch.

Reply:
@romseygeek  Thanks Alan, I've upgraded to the new lucene snapshot and removed `XUkrainianMorfologikAnalyzer` in favour of Lucene's one  in ad97497abfd